app/controllers/api/v1/venues_controller.rb

```diff
@@ -0,0 +1,57 @@
+# :reek:InstanceVariableAssumption
+# rubocop:disable Style/ClassAndModuleChildren
+class Api::V1::VenuesController < ApplicationController
+  before_action :authenticate_user!, only: %i[create update destroy]
+  before_action :set_venue, only: %i[show update destroy]
+
+  def index
+    @venues = Venue.all.reverse
```

Comment:
what is the reason for `reverse`?

Reply:
When i add new venue and  after redirect to the index page it shows me my new venue at the top of iteration.

Reply:
use default sorting

Reply:
added scope to venue model
```ruby 
  default_scope { order(created_at: :desc) }
```